de.dashup.shared/src/main/java/de/dashup/shared/DatabaseModels/DatabaseWidget.java

```diff
@@ -0,0 +1,91 @@
+package de.dashup.shared.DatabaseModels;
+
+import com.google.gson.annotations.SerializedName;
+import de.dashup.shared.Enums.Size;
+
+public class DatabaseWidget extends DatabaseObject{
+
+    private Integer id;
+    private String name;
+    private String description;
+    @SerializedName("overview_text")
+    private String overviewText;
+    private String category;
+    @SerializedName("publisher_id")
+    private Integer publisherID;
```

Comment:
Technically this is a user; we don't have a publisher. Even though a user can publish a widget, a user is not a publisher.

Reply:
I just thought it was confusing having an attribute "id" and another called "user_id" in the user table when the user_id is the publisher_id.